collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -2105,6 +2105,8 @@ public void loadingComplete(FormLoaderTask task, FormDef formDef, String warning
         final FormController formController = task.getFormController();
 
         if (formController != null) {
+            Timber.w("Form %s started", formController.getFormTitle());
```

Comment:
Can we hash this please? I know this kind of info sometimes shows up in crash logs but when we're explicitly logging I think we should hash.

Reply:
Ok we can do that.